server/graphql/v2/object/Activity.ts

```diff
@@ -52,11 +52,11 @@ export const Activity = new GraphQLObjectType({
       type: new GraphQLNonNull(GraphQLJSON),
       description: 'Data attached to this activity (if any)',
       resolve(activity): object {
+        const toPick = ['error'];
```

Comment:
Are we 100% that the error message can't contain sensitive or private info, such as accounts IDs or tokens?

Reply:
If we're not sure, I think we should pick this field only if `remoteUser` is a host admin.

Reply:
Well, this is out of our control so there's no way to be certain. I'll validate `remoteUser` is host admin.